config/app_config.yml.sample

```
@@ -764,7 +764,7 @@ defaults: &defaults
         isolines: false
         routing: false
         data_observatory: false
-
+  disable_email_mx_check: # true / false (default). Disables MX DNS record check and just performs syntax validation.
```

Maybe it's easier to understand this way:
```suggestion
  disable_email_mx_check: false # Disables MX DNS record check and just performs syntax validation.
```